microsoft/testsuites/core/floppy.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import RemoteNode, TestCaseMetadata, TestSuite, TestSuiteMetadata
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite ensures the floppy module is disabled.
```

Comment:
Add reference to let users know, why the floppy module needs to be disabled. And it's good to introduce how to detect the floppy is disabled, like from lsmod.